app/models/shelter.rb

```diff
@@ -1,6 +1,6 @@
 class Shelter < ApplicationRecord
   has_many :pets, dependent: :destroy
-  has_many :reviews
+  has_many :reviews, dependent: :destroy
```

Comment:
When you get this error
```rails aborted! ActiveRecord::InvalidForeignKey: PG::ForeignKeyViolation: ERROR: update or delete on table "users" violates foreign key constraint "fk_rails_03de2dc08c" on table "comments" DETAIL: Key (id)=(11) is still referenced from table "comments". : DELETE FROM "users" WHERE "users"."id" = $1```

put dependent: :destroy so that a shelter can be deleted even if there are reviews and pets still present